lib/monitor/monitor.go

```diff
@@ -18,31 +18,51 @@ import (
 var Sink = "0.0.0.0"
```

Comment:
Each struct has some comments before its definition that describe what they are and what they do related to the others structs. Does this seem reasonable ? What is missing ?